### PR TITLE
Created the login page

### PR DESCRIPTION
**Pull request by HubertasVin**
Added Login/Register page and modified the readme page to include how to install dotnet-ef

### CODE REVIEW

File: Project/ClientApp/src/components/Login.js [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+    }
+  };
+
+  const validateEmail = (email) => {
+    return String(email)
+      .toLowerCase()
+      .match(
+        /^(([^<>()[\]\\.,;:\s@"]+(\.[^<>()[\]\\.,;:\s@"]+)*)|.(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/
+      );
+  };
```
Comment (Sc0rpie) [inline Project/ClientApp/src/components/Login.js]: Regex validation should be done in C# (can make helper folder that will have helper classes. Example: RegexValidation.cs that has function with regex to validate for example an email)

**Review by Sc0rpie — CHANGES_REQUESTED** [review]
Make regex checks in C# instead of JavaScript

File: Project/ClientApp/src/components/Login.js (hunk 1 above)
Comment (HubertasVin) [reply Project/ClientApp/src/components/Login.js]: Won't that not allow to print an error message to the user if the email is invalid (for example a javascript alert)?

Comment (Sc0rpie) [reply Project/ClientApp/src/components/Login.js]: You are able to create your own OK or BadRequest with data inside of it, for example if the request was successful or not and a message
`return Ok(new {success = true, message = "Logged in"`
or
`return BadRequest(new {success = false, message = "Invalid email or password"`

Depending on the result you could write something about bad email or password

Comment (HubertasVin) [reply Project/ClientApp/src/components/Login.js]: Ok, I'll get onto that later

**Pull request merged**